libs/wingc/src/type_check.rs

```diff
@@ -624,6 +624,17 @@ impl TypeRef {
 			_ => false,
 		}
 	}
+
+	/// Returns `true` if the type is a resource or a collection of resources
+	pub fn contains_resources(&self) -> bool {
```

Comment:
Should structs or interfaces be included here? (I'm not sure)

Reply:
Getting rid of this.